orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -173,6 +175,9 @@ public class SaltOrchestrator implements HostOrchestrator {
     @Value("${cb.max.salt.cloudstorage.validation.retry:3}")
     private int maxCloudStorageValidationRetry;
 
+    @Value("${freeipa.delayed.scale-sec:1}")
```

Comment:
You have defaulted it to 1 sec here, but not in the FreeIpaUpscaleActions class. Is there a reason?

Reply:
this service is injected outside of FMS too, so I needed a default value, but the method is not used there, only from FMS where this value is coming from app yaml